resources/js/details.js

```diff
@@ -0,0 +1,9 @@
+function addToShoppingCart(element){
+    //Get the Text of the <p> Element of the List where the button was clicked.
+    let text = element.previousElementSibling.innerText;
+    console.log(text);
+    var cart = JSON.parse(localStorage.getItem("cart"));
+    if(cart == null) cart = [];
+    cart.push(text);
```

Comment:
After you switch to storing IDs, you should check whether an ingredient is already in the shopping cart.

Reply:
I implemented a changes to the remove button, so no issue will arise